source/technical/daypass.rst

```diff
@@ -0,0 +1,91 @@
+.. _daypass:
+
+=======
+Daypass
+=======
+Normally, only Chameleon users with active allocations are able to launch and
+view Trovi artifacts. To allow anyone to launch an artifact, we also provide
+daypass. This allows for a non-Chameleon user to have access to Chameleon
+for a limited amount of time, using a small, separate allocation.
+People interested in reproducing your project will send requests
+to the managers of a project. If approved, the requesting user will recieve an
+email invitation to join a reproducibility project. When they accept, they
+can use this project to run your artifact. After the specified time limit,
+they will be automatically removed from this project.
+
+.. _enable-daypass:
+
+Allowing Reproducibility Requests
+=================================
+
+First, the owner of an artifact must permit reproducibility requests. This can
```

Comment:
I see that we say this here but good to call it out in the opening sentences too I think.